packages/components-react/projects/react-wrapper/tests/unit/specs/usePrefix.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { skipCheckForPorscheDesignSystemProviderDuringTests, usePrefix } from '../../../src/hooks';
```

Comment:
Separate test file implies that utility is in a separate file, too, but that's not the case.

Reply:
Thats why i added a comment to the top of the file about the react mock which influences the whole test file. Would you split the usePrefix() into a seperate file?

Reply:
Try to move files together, otherwise split usePrefix in seperate file